contrib/charts/cert-manager/templates/hook_create_issuers.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if .Values.clusterIssuer.email }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: create-issuers
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/hook-delete-policy": hook-succeeded
```

Comment:
Should this also be a post-upgrade hook? What if a user changes the ACME server URL after deploying?

Reply:
Good catch, I renamed this file with using apply instead of create.

Reply:
Actually reverted to specifically not udpate. Will expalin later, in a GitHub comment of this PR.